libs/foundry-dev-tools/src/foundry_dev_tools/clients/compass.py

```diff
@@ -427,7 +430,7 @@ def api_process_marking(
         self,
         rid: api_types.Rid,
         marking_id: api_types.MarkingId,
-        path_operation_type: api_types.PatchOperation | str,
```

Comment:
I actually wanted to use the former PatchOperation object in TypedDict for API models. But because the former PatchOperation was a enum class, the enum type would not resolve correctly in the dict (instead of 'ADD', the result was 'PatchOperation.ADD'). So I renamed it to PatchOperationType and introduced a new PatchOperation object of type Literal to be used in the API object classes extending TypedDict.
I didn't know if there was a more elegant way to solve it (and reduce redundancy as both PatchOperation and PatchOperationType basically represent the same).

Extract from [api_types.py](https://github.com/emdgroup/foundry-dev-tools/blob/v2/libs/foundry-dev-tools/src/foundry_dev_tools/utils/api_types.py):

```
PatchOperation = Literal["ADD", "REMOVE"]
"""Foundry PatchOperation enum."""


class PatchOperationType(str, Enum, metaclass=EnumContainsMeta):
    """Foundry patch operations for markings."""

    ADD = "ADD"
    """Add marking"""
    REMOVE = "REMOVE"
    """Remove marking"""

    def __str__(self) -> str:
        return self.value
```